service/jobs_service.go

```
@@ -125,8 +125,15 @@ func (s *JobService) CreateRecordingJobDocker(cfg JobConfig, onStopCb stopCb) (J
 		return Job{}, fmt.Errorf("failed to list containers: %w", err)
 	}
 
+	devMode := os.Getenv("DEV_MODE")
+
 	if len(containers) >= s.cfg.MaxConcurrentJobs {
-		return Job{}, fmt.Errorf("max concurrent jobs reached")
+		if devMode == "true" {
```

We may be able to avoid the else block if we reverse the check and fail early if not in dev mode.

---

Yah, I had that at first, but then I thought "avoid negative checks" -- it's been drilled in to me. But happy to make it simpler.

---

Yeah, personally I prefer it in case the negative is on a single variable because it's pretty simple to understand.